walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -16,12 +16,18 @@
 
 import java.util.Collection;
 import java.util.ArrayList;
+import java.util.Stack;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    ArrayList<TimeRange> busyIntervals = new ArrayList<TimeRange>();
-    ArrayList<TimeRange> freeIntervals = new ArrayList<TimeRange>();
+    /**
+     * importantRanges - store time ranges that are important for at leas on attendee in the request
+     * busyIntervals - eventualy store merged important time ranges
+     * freeIntervals - used to collect intevals that are between busyIntervals   
+     */
```

Comment:
Thanks for adding comments. Some typos to fix:
* leas -> least
* on -> one
* eventualy -> eventually